### DISCUSSION TITLE
Feature: Touch face animation?

### DISCUSSION THREAD

**Discussion by snowleopardd** [xesdoog/YimActions#35] [Ideas]
Hey i know this post sounds weird, but i've encountered some modders that would raise their both hands to my face level and move them like they were touching my face, it was pretty cool, but it's a pain to look for the animation when there's like 50k anims, if anyone knows what anim it is would be amazing :)

**Comment by xesdoog** [comment]
I haven't seen anyone doing that animation but from what you've described, this could be a maid cleaning anim or something related to Chris Formage and the Epsilon cult. I'll see if I can find it next time I play the game.
